app/controllers/comments_controller.rb

```diff
@@ -1,5 +1,6 @@
 class CommentsController < ApplicationController
   before_action :require_login
+  before_action :correct_user, only: [:destroy]
```

Comment:
Small detail, but I try to name `before_filters` with a bit more detail about their purpose. This is ok, but I would likely go with something like `require_owner` or similar. "correct" is vague in this context, whereas "owner" makes clear what you are checking without needing to read the method.

To be clear, this is not a problem at all and your code will work just fine, but picking good names and organizing code for ease of reading and understanding turns out to be something of the most valuable and meaningful work a developer does, so I'll likely be commenting on naming and the like more as we step up your game.